config/observability.go

```diff
@@ -54,8 +56,21 @@ func configureTracing(serviceCtx context.Context, logger *zap.Logger) error {
 		shutdownCtx, cancel := context.WithTimeout(context.Background(), time.Second*2)
 		defer cancel()
 		err := traceExporter.Shutdown(shutdownCtx)
-		logger.Error("Failed to flush telemetry data", zap.Error(err))
+		slog.Default().ErrorContext(serviceCtx, "Failed to flush telemetry data", slog.Any("error", err))
 	}(serviceCtx)
 
 	return nil
 }
+
+func configureLogging() error {
+	var err error
+	var logger *slog.Logger
+	if viper.GetBool("JSON_LOGS") {
+		logger = otelslog.NewLogger("openai-discord-bot")
```

Comment:
json logs are synonymous with otel logs now, which feels pretty reasonable to me in this context

Reply:
I ended up moving to supporting both json local logs, and otlp logs in 820b6f3